src/main/java/org/jitsi/jicofo/ChatRoomRoleAndPresence.java

```diff
@@ -209,7 +209,7 @@ private void electNewOwner()
             {
                 continue;
             }
-            else if (MemberRole.OWNER.compareTo(member.getRole()) >=0)
+            else if (member.getRole() == MemberRole.OWNER)
```

Comment:
This assumes owner is the highest role?  Would it be more symmetric to have a `hasOwnerRights()` method?